docs/source/azure/examples/appserviceplansresize-compute.rst

```diff
@@ -1,7 +1,8 @@
 .. _azure_examples_resize_app_service_plan:
 
-Resize an Application Service Plan
-==============================================================
+App Service - Resize an Application Service Plan
```

Comment:
@tylergibson I don't quite the naming structure of the files:
"appserviceplanresize-compute.rst"
"storageremovepublicaps-networking-storage.rst"

Are you just appending the potential sections at the end of the file name?

Reply:
Yes, take a look here: https://github.com/cloud-custodian/cloud-custodian/pull/4316/files#diff-0b9b885cddd1f625f494249158afad78R48

Index includes all examples based on their names.. Networking group == examples where file name contains 'networking' keyvord.